src/JustSaying/JustSayingSerializationContext.cs

```diff
@@ -0,0 +1,12 @@
+﻿#if NET8_0_OR_GREATER
+using System.Text.Json.Serialization;
+using JustSaying.AwsTools.QueueCreation;
+using JustSaying.Messaging.MessageSerialization;
+
+namespace JustSaying;
+
+[JsonSerializable(typeof(SqsMessageEnvelope))]
+[JsonSerializable(typeof(IReadOnlyCollection<string>))]
+[JsonSerializable(typeof(RedrivePolicy))]
+internal sealed partial class JustSayingSerializationContext : JsonSerializerContext;
```

Comment:
It's never occured to me to use `;` than just have an empty `{...}` body instead 👍